tracing/src/macros.rs

```diff
@@ -783,6 +783,123 @@ macro_rules! event {
     );
 }
 
+/// Checks whether a span or event is enabled depending on the provided metadata.
```

Comment:
nit: maybe worth linking to the `enabled` method here? not a big deal
```suggestion
/// Checks whether a span or event is [enabled] based on the provided [metadata].
///
/// [enabled]: crate::Collect::enabled
/// [metadata]: crate::Metadata
```